panel/core/esm.py

```diff
@@ -0,0 +1,148 @@
+from __future__ import annotations
```

Comment:
Why is the `esm.py` file there twice?

Reply:
Was playing with adding a core subpackage but backed out of that.